### PR TITLE
Feature/user deleted event

### PR DESCRIPTION
**Pull request by imtoori**

### CODE REVIEW

**Comment by jaapbakker88** [comment]
https://github.com/GetStream/chat/pull/1060

**Comment by ferhatelmas** [comment]
@jaapbakker88 how about types?

**Comment by jaapbakker88** [comment]
> @jaapbakker88 how about types?

don't think types are an issue here.

File: src/events.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ export const EVENT_MAP = {
 	// local events
 	'connection.changed': true,
 	'connection.recovered': true,
+	'user.deleted': true,
```
Comment (ferhatelmas) [inline src/events.js:38]: is this location in the map, right ?

Comment (ferhatelmas) [reply src/events.js]: ordered

**Pull request merged**